src/frontend/admin/src/services/api/models/Order.ts

```diff
@@ -30,7 +30,7 @@ export type Order = AbstractOrder & {
   order_group?: OrderGroup;
   course?: Course;
   enrollment?: Enrollment;
-  certificate: {
+  certificate?: {
```

Comment:
This is not optional, it is nullable.
This comment can be applied to other optional fields.

```suggestion
  certificate: Nullable<{
```